docs/docsite/rst/scenario_guides/guide_docker.rst

```diff
@@ -30,12 +30,12 @@ Ansible 2.1.0 includes major updates to the Docker modules, marking the start of
 integrated set of tools for orchestrating containers. In addition to the above modules, we are also working on the
 following:
 
-Still using Dockerfile to build images? Check out `ansible-container <https://github.com/ansible/ansible-container>`_,
+Still using Dockerfile to build images? Check out `ansible-bender <https://github.com/ansible-community/ansible-bender>`_,
 and start building images from your Ansible playbooks.
 
-Use the *shipit* command in `ansible-container <https://github.com/ansible/ansible-container>`_
+Use `Ansible Operators <https://learn.openshift.com/ansibleop/ansible-operator-overview/>`_
```

Comment:
From how I understand the website, `Operator` should be singular:
```suggestion
Use `Ansible Operator <https://learn.openshift.com/ansibleop/ansible-operator-overview/>`_
```
There are also a lot of `Ansible Operators` usages all over the web, but the official sites always seem to say `Ansible Operator`. Do you have a better insight into the naming?

Reply:
Nope, I don't have specific insight into the naming... but after re-reading with fresh eyes, it does look like "the Ansible Operator" is the official way from OpenShift's docs. Since we agree, I'll update to standardize on that.